modules/admediaBidAdapter.js

```diff
@@ -11,10 +11,29 @@ var adaptermanager = require('src/adaptermanager');
  *
  */
 var AdmediaAdapter = function AdmediaAdapter() {
+  let uid = '';
+  let bids;
+  const bidderUrl = (window.location.protocol) + '//b.admedia.com/banner/prebid/bidder/?';
+
   function _callBids(params) {
-    var bids;
-    const bidderUrl = (window.location.protocol) + '//b.admedia.com/banner/prebid/bidder/?';
     bids = params.bids || [];
+    getUID();
+  }
+
+  function getUID() {
+    var xhttp = new XMLHttpRequest();
+    xhttp.onreadystatechange = function() {
+      if (this.readyState == 4 && this.status == 200) {
+        uid = this.responseText;
+        generateBidRequests();
+      }
+    };
+    xhttp.withCredentials = true;
+    xhttp.open('GET', (window.location.protocol) + '//pixel.s3xified.com/uid', true);
```

Comment:
we don't allow pixel requests to hold up the auction. This will need to be moved to the cookie sync module.

Reply:
Submitted PR #1923 with suggested changes.